WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsOnboardingError.swift

```diff
@@ -13,34 +13,36 @@ struct InPersonPaymentsOnboardingError: View {
     var secondaryButtonViewModel: InPersonPaymentsOnboardingErrorButtonViewModel? = nil
 
     var body: some View {
-        VStack {
-            Spacer()
+        ScrollableVStack(padding: 0) {
+            VStack {
```

Comment:
Nit: Correct me if I'm wrong but I think we don't need this VStack anymore, right? Since the ScrollableVstack already wraps the content in one. I quickly removed it and re-tested the changes and seems to have no effect.

Reply:
Good call, I tried removing the VStack and just use the VStack in `ScrollableVStack`. However, the padding of the VStack in `ScrollableVStack` does not accept a nil value for iOS to use the system padding (so that it works like `.padding()`). To avoid unexpected regressions on the layout, I think it's best to keep the original `.padding()` on the error content.